src/Utility/Composer.php

```diff
@@ -36,17 +36,22 @@ public static function getLibraryVersions(CommandRunner $runner, $library)
      * @param string $version
      * @param string $repository Optional custom repository
      * @param bool $preferDist Set to true to use dist
+     * @param bool $ignorePlatform Set to true to ignore platform deps
      */
     public static function createProject(
         CommandRunner $runner,
         $recipe,
         $directory,
         $version,
         $repository = null,
-        $preferDist = false
+        $preferDist = false,
+        $ignorePlatform = false
     ) {
+        // Flush cache prior to create; Otherwise recent tags may not be present.
+        $runner->runCommand(['composer', 'clear-cache']);
```

Comment:
Curious - for me on command line this didn't help, but `git fetch --tags` did

Reply:
Yeah this bug may need fixing again later.